funnel/views/project.py

```diff
@@ -213,6 +240,7 @@ def session_videos(self):
             'project': self.obj,
             'cfp_transition_form': cfp_transition_form,
             'project_save_form': project_save_form,
+            'empty_form': forms.Form(),
```

Comment:
So far we've been calling these `csrf_form`. Keep the same or change all?

Reply:
Changed to `csrf_form`